README.md

```diff
@@ -41,13 +34,10 @@ var token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJjbGFpbTEiOjAsImNsYWltMiI6Im
 var secret = "GQDstcKsx0NHjPOuXOYg5MbeJ1XT0uFiwDVvVBrk";
 try
 {
-    IJsonSerializer serializer = new JsonNetSerializer();
-    IDateTimeProvider provider = new UtcDateTimeProvider();
-    IJwtValidator validator = new JwtValidator(serializer, provider);
-    IBase64UrlEncoder urlEncoder = new JwtBase64UrlEncoder();
-    IJwtDecoder decoder = new JwtDecoder(serializer, validator, urlEncoder);
-    
-    var json = decoder.Decode(token, secret, verify: true);
+    var json = new JWTBuilder()
+        .SetSecret(secret)
+        .MustVerify()

```

Comment:
nitpick: I think `RequireVerification` is clearer

Reply:
Changed to MustVerifySignature